docs/guide/areas-and-widgets/README.md

```diff
@@ -0,0 +1,137 @@
+
+### Leave `-widget` out of area configuration
+
+If you dig into the Apostrophe code, or read the guide to custom widget types, you may already know that widget module names end with `-widget`, such as `@apostrophecms/rich-text-widget`. This suffix is left off when configuring areas. The area knows these are widgets already and it does not seem necessary to require developers don't need to write it over and over.
+
+### Limiting the number of widgets in an area
+
+To limit the number of widgets allowed in a specific area, include a `max` option.
+
+```javascript
+main: {
+  type: 'area',
+  options: {
+    widgets: {
+      '@apostrophecms/rich-text': {},
+      '@apostrophecms/image': {},
+      '@apostrophecms/video': {}
+    },
+    max: 3
+  }
+}
+```
+
+This can be especially useful when you want to allow only one widget of a particular type in an area. This can be a good way to add a special page introduction that should only include a single rich text area.
+
+```javascript
+introduction: {
+  type: 'area',
+  options: {
+    widgets: {
+      '@apostrophecms/rich-text': {}
+    },
+    max: 1
+  }
+}
+```
+
+## Adding areas to templates
+
+Areas have a special template tag to add them in template markup. It requires passing two arguments: the area's context and the area name.
+
+```django
+{% area context, name %}
+```
+
+The **context** refers to the data object that the area field belongs to. This could be the a page, a piece, or a widget. In the landing page example above, the `main` area belongs to a landing page. In that case, the context would be that page's data object in the template: `data.page`.
+
+```django
+{% area data.page, main %}
+```
+
+The template tag knows to use the area data on `data.page.main`, check for the widgets allowed in that area, and render the area using the correct widget templates.
+
+### Passing context options
+
+Most widget options must be included in the area field configuration. For example, you can configure the rich text widget to use particular formatting controls. The server uses these options to properly validate user input.
+
+```javascript
+main: {
+  type: 'area',
+  options: {
+    widgets: {
+      '@apostrophecms/rich-text': {
+        toolbar: [
+          'bold',
+          'italic',
+          'strike',
+          'link'
+        ]
+      }
+    }
+  }
+}
+```
+
+::: note
+Learn more about rich text options in [the section on core widgets](/guide/areas-and-widgets/core-widgets.md).
+:::
+
+In other situations, you may need to **pass the widget *template* options that only apply to a specific context** and not everwhere the area is shown. One example of this is the `sizes` attribute for the core image widget's `img` tag. Since that attribute tells browsers which file versions to use in a responsive image, it may be different when the image is a small thumbnail as opposed to when it is a larger featured photo.
```

Comment:
```suggestion
In other situations, you may need to **pass the widget *template* options that only apply to a specific context**. One example of this is the `sizes` attribute for the core image widget's `img` tag. Since that attribute tells browsers which file versions to use in a responsive image, it may be different when the image is a small thumbnail as opposed to when it is a larger featured photo.
```